src/components/properties/PropertyCard.tsx

```diff
@@ -31,18 +33,42 @@ const PropertyCardContents = (props: PropertyCardProps) => {
     }, [property_id]);
 
     return (
-        <PropertyCardContainer
-            onClick={() => {
-                navigate("/property/" + property_id);
-            }}
-        >
-            <PropertyImage src={image_url} />
-            <CardTitle>
-                {address}
-            </CardTitle>
-            <CardText>
-                {tasks.length} tasks
-            </CardText>
+        <PropertyCardContainer>
+            <PropertyImageWrapper>
+                <PropertyImage src={image_url}
+                    onClick={() => {
+                        navigate("/property/" + property_id);
+                    }}
+                    onKeyDown={(event: React.KeyboardEvent) => {
+                        if (event.key === 'Enter') {
+                            navigate("/property/" + property_id);
+                        }
+                    }}
```

Comment:
sonarcloud got mad because it said we can't have an onClick for an invisible component unless we also had an onKeyDown, for accessibility readers. I can make the function for this empty if that's better

Reply:
make whole thing a button, increase zAxis of edit button